### PR TITLE
feat: [CDS-34155]: test coverage for ManifestSource

### PR DESCRIPTION
**Pull request by Tamaraius**
##### Summary:

Test cases for ManifestSource

##### Jira Links:

https://harness.atlassian.net/browse/CDS-34155

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by YounixIO** [comment]
retrigger coverage

**Comment by Tamaraius** [comment]
retrigger coverage

File: src/modules/75-cd/factory/ManifestSourceFactory/ManifestSourceBaseFactory.tsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ import { OpenshiftTemplateManifestSource } from '@cd/components/PipelineSteps/K8
...
-import { HelmGitlabManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmGitlabManifestSource'
-import { HelmGithubManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmGithubManifestSource'
+import { HelmGitManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmGitManifestSource/HelmGitManifestSource'
+import { HelmBitbucketManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmBitbucketManifestSource/HelmBitbucketManifestSource'
+import { HelmGitlabManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmGitlabManifestSource/HelmGitlabManifestSource'
+import { HelmGithubManifestSource } from '@cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmGitManifestSource/HelmGithubManifestSource/HelmGithubManifestSource'
```
Comment (ishantmehtaharness) [inline src/modules/75-cd/factory/ManifestSourceFactory/ManifestSourceBaseFactory.tsx:17]: Any particular reason to change these location?

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good with this comment
https://github.com/harness/harness-core-ui/pull/7504/files#r814937095

File: src/modules/75-cd/factory/ManifestSourceFactory/ManifestSourceBaseFactory.tsx (hunk 1 above)
Comment (bhavya-sinha) [reply src/modules/75-cd/factory/ManifestSourceFactory/ManifestSourceBaseFactory.tsx:17]: This is to add test case in same folder. The path has been changed

**Review by bhavya-sinha — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
LGTM

**Review by vkbansal — APPROVED** [review]

**Pull request merged**